cinema/serializers.py

```diff
@@ -1 +1,84 @@
-# write serializers here
+from rest_framework import serializers
+
+from cinema.models import Genre, Actor, Movie, MovieSession, CinemaHall
+
+
+class GenreSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Genre
+        fields = ("id", "name")
```

Comment:
Consider adding type annotations to the `Meta` class methods to improve code readability and maintainability. This is a requirement according to the task checklist.